packages/components/src/components/buttons/Button/index.tsx

```diff
@@ -36,8 +36,8 @@ const Wrapper = styled.button<WrapperProps>`
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
     color: ${props => (props.color ? props.color : colors.BLACK25)};
     outline: none;
-    padding: ${props => (props.variant === 'tertiary' ? '0px 4px' : BUTTON_PADDING[props.size])};
-    height: ${props => (props.variant === 'tertiary' ? '20px' : 'auto')};
+    padding: ${props => BUTTON_PADDING[props.size]};
+    height: 'auto';
```

Comment:
nitpick: quotes are not necccessaarryryy

Reply:
huh, actually whole line could be deleted, since this is the default behaviour right?